pkg/management/component.go

```diff
@@ -506,6 +508,32 @@ func (c *component) GetGroupsOfUser(ctx context.Context, realmName, userID strin
 	return groupsRep, nil
 }
 
+func (c *component) GetAvailableTrustIDGroups(ctx context.Context, realmName string) ([]string, error) {
+	var res []string
+	for key := range c.authorizedTrustIDGroups {
+		res = append(res, key)
+	}
+	return res, nil
+}
+
+func (c *component) GetTrustIDGroups(ctx context.Context, realmName, userID string) ([]string, error) {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+	var currentUser, err = c.keycloakClient.GetUser(accessToken, realmName, userID)
+	if err != nil {
+		c.logger.Warn(ctx, "err", err.Error())
+		return nil, err
+	}
+	var groups = make([]string, 0)
+	for _, grp := range currentUser.GetAttribute(constants.AttrbTrustIDGroups) {
+		if strings.HasPrefix(grp, "/") {
```

Comment:
out of curiosity: some groups are written with / as preffix ?

Reply:
In fact, you should always have a leading /. If you look at method SetTrustIDGroups, you'll see that we always add a / as a prefix. I'm checking if the prefix is here in case of bad values in the attributes but we could potentially remove this test.
(that is also the default behavior of Keycloak user groups)

Reply:
thanks for the explanation